### PR TITLE
 update env file and add api key

### PR DESCRIPTION
**Pull request by rickDevelopment**
Update env file

### CODE REVIEW

File: project/.env-sample [hunk 1]
```diff
@@ -1 +0,0 @@
-API_KEY=YOUR_API_KEY
```
Comment (fesebuv) [inline project/.env-sample:1]: I thought you updated this value?

**Review by fesebuv — APPROVED** [review]

File: project/src/public/client.js [hunk 2]
```diff
@@ -103,3 +103,4 @@ const getImageOfTheDay = (state) => {
 
     return data
 }
+// Hey check this out
```
Comment (fesebuv) [inline project/src/public/client.js]: 👍

File: project/src/public/client.js [hunk 3]
```diff
@@ -103,3 +105,5 @@ const getImageOfTheDay = (state) => {
...
+
```
Comment (fesebuv) [inline project/src/public/client.js:109]: ok!

**Comment by UAnjali** [comment]
Hi,
We are closing out the issue because there are no enhancements proposed here.
Thank you

**Pull request closed without merging**